browser-extension/package.json

```diff
@@ -17,6 +17,7 @@
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "axios": "^1.4.0",
+    "axios-case-converter": "^1.1.1",
```

Comment:
This will convert between snake case (used by api) and camel case (used by frontend) JSON keys